src/xdocs/property_types.xml

```diff
@@ -709,15 +709,19 @@
             <td><code>always</code></td>
             <td>
               Example:
-              <pre>@Deprecated()</pre>
+              <div class="wrapper">
+                <pre>@Deprecated()</pre>
```

Comment:
Why was this changed? This isn't an image or long text. Is this just because of the `pre`? If so, do we need a junit to verify the `pre`s too?

Reply:
yeah, this has to be wrapped, too. There was a bug in the unit test which is mentioned in the pr description, which causes only the config_*.xml files to be checked. this is one of the fixes which had to be done after fixing the test.

`pre` is a tag which is displayed in html, `source` gets rendered to a `pre` in a `div class="source"`. I don't know the exact difference and why sometimes a `source` and sometimes a `pre` is used tough.

Reply:
ok. Thanks.